src/software/pipeline/main_LdrToHdrMerge.cpp

```diff
@@ -175,7 +175,29 @@ int aliceVision_main(int argc, char** argv)
         }
     }
     std::vector<std::shared_ptr<sfmData::View>> targetViews;
-    hdr::selectTargetViews(targetViews, groupedViews, offsetRefBracketIndex);
+    //hdr::selectTargetViews(targetViews, groupedViews, offsetRefBracketIndex);
+
+    const std::string targetIndexFilename = (fs::path(inputResponsePath).parent_path() / (std::string("exposureRefIndexes") + std::string(".txt"))).string();
+
+    std::ifstream file(targetIndexFilename);
+    std::vector<int> targetIndexes;
+    if (!file)
+    {
+        throw std::logic_error("Can't open target indexes file");
+    }
```

Comment:
```suggestion
    const std::string targetIndexFilepath(fs::path(inputResponsePath).parent_path() / (std::string("exposureRefIndexes.txt")));

    std::ifstream file(targetIndexFilepath.string());
    std::vector<int> targetIndexes;
    if (!file)
    {
        ALICEVISION_THROW_ERROR("Unable to open the file " << targetIndexFilepath << " with the selection of exposures. This file is needed to select the optimal exposure for the creation of the HDR images.");
    }
```

Reply:
WARNING instead of ERROR because the clamped parameter offsetRefBracketIndex is used as a fallback solution in this case.